core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.ts

```diff
@@ -62,7 +65,11 @@ export class ContributorDashboardAdminNavbarComponent implements OnInit {
   async getUserInfoAsync(): Promise<void> {
     const userInfo = await this.userService.getUserInfoAsync();
 
-    this.username = userInfo.getUsername();
+    let userName = userInfo.getUsername();
```

Comment:
Prefer `const` here. It also seems odd to create a separate username variable with this capitalization. If you try to assign the potential null username to `this.username` directly here does that raise an error?